src/wizard/deploy.jl

```diff
@@ -25,7 +25,7 @@ function print_build_tarballs(io::IO, state::WizardState)
         """
     end
 
-    if state.files === nothing
+    if state.files === nothing || state.file_kinds === nothing || state.file_varnames === nothing
```

Comment:
This is a bit shorter:
```suggestion
    if any(isnothing, (state.files, state.file_kinds, state.file_varnames))
```